include/uapi/asm-generic/unistd.h

```diff
@@ -854,7 +854,7 @@ __SYSCALL(__NR_clone3, sys_clone3)
 __SYSCALL(__NR_pidfd_mem, sys_pidfd_mem)
 
 #undef __NR_syscalls
-#define __NR_syscalls 436
+#define __NR_syscalls 999
```

Comment:
Ooops! I think 1000 is the right value, if 999 is the last syscall number.

Reply:
You are completely right, my bad, sorry!

Reply:
Same issue was in the QEMU PR, which I just updated to 1000 as well.